otx/cli/tools/train.py

```diff
@@ -175,7 +175,7 @@ def main():  # pylint: disable=too-many-branches
             "ann_file": data_config["data"]["val"]["ann-files"],
             "data_root": data_config["data"]["val"]["data-roots"],
         }
-    if data_config["data"]["unlabeled"]["data-roots"]:
+    if "unlabeled" in data_config["data"] and data_config["data"]["unlabeled"]["data-roots"]:
```

Comment:
The objective of this line is to avoid the error caused when "unlabeled" is not in `data_config["data"]`?
IMO, If there is no "unlabeled" key in `data_config["data"]`, this line still makes an error.

Reply:
It will not raise an error since we have "lazy" if in Python.